src/sections/Company/Cloud-Native/cloud-native.style.js

```diff
@@ -57,10 +57,10 @@ const CloudNativeWrapper = styled.div`
 
     }
     .btn-cont button{
-        color:${props => props.theme.DarkTheme ? "black" : "white"};
+        color:${props => props.theme.DarkTheme ? "black" : "black"};
```

Comment:
Well, if we are using the same color for both dark/light themes then simply remove the conditional statement and use the respective color.
Update at other places as well, that you have changed in this PR.